app/models/form/lettings/questions/renewal.rb

```diff
@@ -3,15 +3,32 @@ def initialize(id, hsh, page)
     super
     @id = "renewal"
     @check_answer_label = "Property renewal"
-    @header = "Is this letting a renewal?"
+    @header = header_text
     @type = "radio"
     @answer_options = ANSWER_OPTIONS
-    @hint_text = "A renewal is a letting to the same tenant in the same property. If the property was previously being used as temporary accommodation, then answer 'no'"
+    @hint_text = hint_text
     @question_number = 4
   end
 
   ANSWER_OPTIONS = {
     "1" => { "value" => "Yes" },
     "0" => { "value" => "No" },
-  }.freeze
+  }
+  .freeze
+
+  def header_text
+    if form.start_date && form.start_date.year >= 2024
```

Comment:
think it's worth saving this as a method as we're going to use this exact thing loads?

Reply:
or can we use `&.`?

Reply:
`&.` will still execute `>=` and it doesn't know what to do in `nil` situations. I've extracted it to form tho, let me know if that's better